test/e2e/common/types.py

```diff
@@ -0,0 +1 @@
+CLUSTER_RESOURCE_PLURAL = 'clusters'
```

Comment:
This file can be deleted if you are not using it. It's leftover from the EKS tests.